src/VuePhoneNumberInput/index.vue

```diff
@@ -142,10 +143,7 @@
         },
         set (newCountry) {
           this.emitValues({countryCode: newCountry, phoneNumber: this.phoneNumber})
-          if (this.inputFocused) {
-            this.$refs.PhoneNumberInput.$el.querySelector('input').focus()
-          }
-          this.inputFocused = true
+          this.$refs.PhoneNumberInput.$el.querySelector('input').focus()
```

Comment:
This seems to autofocus the element on pageload, when it fetches the current country code.